controllers/routemonitor/routemonitor_test.go

```diff
@@ -161,6 +151,7 @@ var _ = Describe("Routemonitor", func() {
 	Describe("EnsureRouteMonitorAndDependenciesAbsent", func() {
 		BeforeEach(func() {
 			// Arrange
+			ensureBlackBoxExporterResourcesAbsentResponse = utilreconcile.Result{Continue: true}
```

Comment:
I used the utilreconcile to create a `ContinueOperation`

Reply:
ack